frontend/src/metabase/account/login-history/components/LoginHistory/LoginHistory.jsx

```diff
@@ -57,6 +62,15 @@ function LoginHistoryList({ loginHistory }) {
   const items = formatItems(loginHistory);
   const groups = _.groupBy(items, item => item.date);
 
+  if (!items || !items.length) {
+    return (
+      <EmptyState
+        title={t`No logins`}
```

Comment:
JFYI @mazameli, this seems to add a new string (for i18n).

Reply:
Thanks for the FYI, Ariya.